psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
I was going to suggest [this](https://github.com/psi4/psi4/blob/2118f2f5b5ffbe66932b8a1f20f4553cd3ee9415/psi4/driver/p4util/procutil.py#L40) for py3, but yes, an `else:` works too, and more simply. Good point, AJ, I'm going to add some tests of this to options1.